api/middleware/authRequired.js

```diff
@@ -1,43 +1,45 @@
-/* istanbul ignore file */
+const jwt = require('express-jwt');
+const jwksRsa = require('jwks-rsa');
 const createError = require('http-errors');
-const OktaJwtVerifier = require('@okta/jwt-verifier');
-const oktaVerifierConfig = require('../../config/okta');
+const Express = require('express');
+const app = new Express();
+
 const Parents = require('../parent/parentModel');
-const oktaJwtVerifier = new OktaJwtVerifier(oktaVerifierConfig.config);
 
-const makeParentObject = (claims) => {
+const makeParentObject = (user) => {
   return {
-    Email: claims.email,
-    Name: claims.name,
+    Name: user.name,
+    Email: user.email,
   };
```

Comment:
I am happy to see that the naming convention was updated to follow along with what we were using on the front end. We were all confused initially when we saw claims. Great catch.